### PR TITLE
Set up message repeating (fixes #20)

### LINKED ISSUE #20: handle repeat nodes
As we talked about, we'll need to figure out a way to do this. We may end up using reselect to process this

### PR DESCRIPTION
**Pull request by bgrins**
This is one approach to implementing message repeating.  We manage this in the store by completely deleting messages out of the array that gets set as state if they are determined to be repeats.  Also set a property on the first message in a set of repeats with the count.  This is nice because it keeps the messages array short and we don't need to re-process known duplicates.

### CODE REVIEW

File: js/components/Message/MessageRepeat.js [hunk 1]
```diff
@@ -2,8 +2,8 @@ import React, { PropTypes } from "react";
 
 function MessageRepeat(props) {
   return (
-    <span value="1" className="message-repeats">
-      // @TODO implement
+    <span value="{props.repeats}" className="message-repeats">
```
Comment (linclark) [inline js/components/Message/MessageRepeat.js]: If this were the final thing, we should have a propTypes definition for repeats, but eh... that can happen later.

**Comment by linclark** [comment]
I wasn't 100% sure of this approach when you were explaining it, but I like it.

There's a conflict between this and one of your earlier PRs. Once that's resolved I can commit it to master.

File: js/reducers/homeReducer.js [hunk 2]
```diff
@@ -10,10 +10,64 @@ function homeReducer(state = initialState, action) {
...
+
+  messages.forEach((message, i) => {
+    // Only try to repeat messages that support this feature
+    if (!message.allowRepeating) {
+      lastRepeatedMessage = lastRepeatID = null;
+      return;
+    }
+
+    // We could be explicit about what we care about in the repeat ID,
+    // or we could just stringify the entire message (sans-timestamp).
```
Comment (linclark) [inline js/reducers/homeReducer.js]: stringifying the entire message sounds good to me. can always change later.

**Comment by bgrins** [comment]
Landed  inhttps://github.com/bgrins/console-output-experiment/commit/b700b0f1f57247f47339162e2a91c175a95e755f

**Pull request closed without merging**